include/padmgr.h

```diff
@@ -19,21 +19,22 @@ typedef struct {
     /* 0x0078 */ UNK_TYPE unk_78;
     /* 0x007C */ UNK_TYPE unk_7C;
     /* 0x0080 */ OSThread thread;
-    /* 0x0230 */ char unk_230[0x78];
-    /* 0x02A8 */ u8 unk_2A8;
-    /* 0x02A9 */ u8 unk_2A9;
-    /* 0x02AA */ u8 unk_2AA[4];
-    /* 0x02AA */ u8 unk_2AE[4]; // Looks like 1 if vibration pack, 2 if (maybe controller pack)?
-    /* 0x02B2 */ u8 unk_2B2[4];
-    /* 0x02B6 */ u8 unk_2B6[4];
-    /* 0x02BA */ char unk_2BA[0x02]; // probably padding
+    /* 0x0230 */ Input inputs[4]; //0x18 each = 0x60 total
```

Comment:
All comments should start with a space, based on the formatting style we went with. There are a few that should be fixed in this header which wasn't touched by clang-format.